webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -962,10 +962,25 @@ func (v validator) validatePowerStateOnUpdate(
 	return allErrs
 }
 
+func cloudInit(vm *vmopv1.VirtualMachine) *vmopv1.VirtualMachineBootstrapCloudInitSpec {
```

Comment:
Maybe change this func to return bool, `IsBootstrapCloudInit(vm) bool`. so later it can check for `true` instead of nil.

Reply:
Done